app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.order(title: :asc).paginate(page: params[:page], per_page: 20)
+    @movies = @movies.filter_by_title(params[:title]) if params[:title].present?
+    
+    if params[:sort].present?
```

Comment:
I don't get this, I thought the possible params where `sort=title` and `sort=release_date` and that the user could also specify the order (`asc` or `desc`). Have you checked if that's possible with this implementation?